kore/test/Test/Kore/Strategies/OnePath/Step.hs

```diff
@@ -229,6 +295,26 @@ test_onePathStrategy =
                     (Mock.functionalConstr10 (TermLike.mkElemVar Mock.y))
                     (Mock.functionalConstr11 (TermLike.mkElemVar Mock.y))
                 ]
+        actualReach <-
```

Comment:
Some of the lines below also go beyond 80 chars; maybe split them over more lines or alias the long function names to a locally short name?